lgc/elfLinker/ElfLinker.cpp

```diff
@@ -140,6 +141,18 @@ class ElfLinkerImpl final : public ElfLinker {
   // -----------------------------------------------------------------------------------------------------------------
   // Implementations of ElfLinker methods exposed to the front-end
 
+  // Add another input ELF to the link, in addition to the ones that were added when the ElfLinker was constructed.
+  // The default behavior of adding extra ones at the start of the list instead of the end is just so you
+  // get the same order of code (VS then FS) when doing a part-pipeline compile as when doing a whole pipeline
+  // compile, to make it easier to test by diff.
+  void addInputElf(MemoryBufferRef inputElf) override { addInputElf(inputElf, /*addAtStart=*/true); }
```

Comment:
Other functions are marked as `final`, should this one and the one below be `final` too?